proxy.py

```diff
@@ -12,6 +12,7 @@
 ports = os.environ["PORT"].split()
 max_connections = os.environ.get("MAX_CONNECTIONS", 100)
 ip = target = os.environ["TARGET"]
+upd_answers = os.environ.get("UDP_ANSWERS", "1")
```

Comment:
```suggestion
udp_answers = os.environ.get("UDP_ANSWERS", "1")
```

Change all code accordingly.

Reply:
Thanks, sorry for using you as a spellcheck.